contrib/src/main/java/com/datatorrent/contrib/hbase/AbstractHBaseAppendOutputOperator.java

```diff
@@ -20,15 +20,17 @@
 
 import java.io.IOException;
 
-import org.apache.hadoop.hbase.client.Append;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import org.apache.hadoop.hbase.client.Append;
```

Comment:
In apex checkstyle, org is listed before org.apache. IDE ordered it accordingly. Maybe the older imports were before the checkstyle configuration was created.